base/db/R/search_references.R

```diff
@@ -32,8 +32,8 @@ search_reference_single <- function(query, limit = 1, min_score = 85) {
     return(tibble::tibble(query = query))
   }
   crdata <- crsearch[["data"]] %>%
-    dplyr::mutate(score = as.numeric(score)) %>%
-    dplyr::filter(score > !!min_score)
+    dplyr::mutate(.data$score = as.numeric(.data$score)) %>%
```

Comment:
`.data` not used on the left side of expressions.

```suggestion
    dplyr::mutate(score = as.numeric(.data$score)) %>%
```